proseco/tests/test_catalog.py

```diff
@@ -69,6 +71,11 @@ def test_get_aca_catalog_20603_with_supplement():
     assert (len(aca_no.acqs) != len(aca.acqs)
             or np.any(aca_no.acqs['mag'] != aca.acqs['mag']))
 
+    assert LooseVersion(aca.supp_agasc_version) > LooseVersion('4.10')
+    assert CxoTime(aca.supp_last_updated) > CxoTime('2021-03-01')
+
+    assert aca_no.supp_agasc_version is None
```

Comment:
For tests I didn't have much in mind, so just threw in checks that the supplement if used was at least as new (with margin) from the one I'm seeing in the 'rc' area.